File: web3rpc/rpc-specs/code-samples/python/personal/lockAccount.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

address = "0xfa415bb3e6231f488ff39eb2897db0ef3636dd32"

w3 = Web3(Web3.HTTPProvider(host))
personal_response = w3.personal.lock_account(address)

print(personal_response)

File: web3rpc/rpc-specs/code-samples/javascript/personal/lockAccount.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");    
    const address = "0xfa415bb3e6231f488ff39eb2897db0ef3636dd32"

    provider.personal.lockAccount(address,{}, (err, data, response) => {
        console.log(data);
    });
}
)()